src/Elsa.Models/InformationText.cs

```diff
@@ -7,5 +7,6 @@ public class InformationText
         public bool IsHyperlink { get; set; } = false;
         public string? Url { get; set; }
         public bool IsBold { get; set; } = false;
+        public bool IsGuidance { get; set; }
```

Comment:
Is this being added back in to deal with the legacy models?  As I think so long as it parses when we're mapping the values, we don't need to carry it through to the front end.

Reply:
discussed and decided to keep it to fully support legacy style guidance in information text